cilium/cilium.io
# feat: update 404 page

This pull request brings update for 404 error page. And also solve issue #376 

**Steps to check:**

- open [preview](https://deploy-preview-390--cilium.netlify.app/error-404)
- make sure that everything looks and feels as expected
